code/controllers/subsystem/ticker.dm

```diff
@@ -351,11 +351,10 @@ SUBSYSTEM_DEF(ticker)
 /datum/controller/subsystem/ticker/proc/station_explosion_detonation(atom/bomb)
 	if(bomb)	//BOOM
 		qdel(bomb)
-		for(var/T in GLOB.station_turfs)
-			if(prob(33))
-				SSexplosions.highturf += T
-			else
-				SSexplosions.medturf += T
+		for(var/mob/M in GLOB.mob_list)
+			var/turf/T = get_turf(M)
+			if(T && is_station_level(T.z) && !istype(M.loc, /obj/structure/closet/secure_closet/freezer)) //protip: freezers protect you from nukes
```

Comment:
That's pretty funny.

Reply:
Indiana Jones reference... this is funny to me.